app/controllers/podcast_planner_controller.rb

```diff
@@ -1,10 +1,15 @@
 class PodcastPlannerController < ApplicationController
   before_action :set_podcast
+  before_action :set_beginning_of_week
 
   # GET /podcasts/1/planner
   def show
     @planner = PodcastPlanner.new(planner_params)
     @planner.generate_dates!
+
+    if @planner.dates.present?
+      @planner_months = @planner.dates_by_month
```

Comment:
could probably just make this its own attribute, but essentially this splits the dates into arrays by month.

Reply:
Is there any way to make the monthly-calendars show up, even if no generator-dates are selected?  So a user can sort of mix/match generating the dates and manually selecting them.

(Maybe this could be a different issue?  No need to clutter this PR)

Reply:
Another way to think about it:

If you decouple the calendar-generating code from the "which days on the calendar are selected" code, it may simplify things?  Just loop through N months starting at `Date.today.beginning_of_month` to generate the calendars.  And for each rendered day, ask "is this selected?"